tasks/install_brotli.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Apache_httpd | Ensure git is installed
+  apt:
+    name: "{{ item }}"
+    update_cache: yes
+    cache_valid_time: 86400
+  with_items:
+    - "{{ apache_brotli_prerrequisites }}"
+
+- name: Apache_httpd | Download brotli
+  git:
+    repo: 'https://github.com/google/brotli.git'
```

Comment:
IMHO that variable should be in defaults.yml

Reply:
Done